load/UpdateDBPTextFilesets.py

```diff
@@ -174,16 +185,16 @@ def getBibleVersesFromSqlite(self, filesetId, databasePath):
 			# NOTE: it is possible that verse value has a format equals to: number + string e.g. "2a".
 			# currently, this format is not allowed but we will need to handle this case.
 			# Also, we should make sure that the order is the correct.
-			verseStart = self.verseString2Int(bookId, chapter, parts[0])
+			verseStart = parts[0]
 			verseEnd = self.verseString2Int(bookId, chapter, parts[len(parts) -1])
```

Comment:
@bradflood , The verseEnd could have a format equals to: number + string e.g. "2a". But, for now on, it will be a number because the method: verseString2Int will cast the string to number. E.g. "2a" => 2

Reply:
I don't see a problem with verse_end remaining as a number. I have not seen any case where verse_end = "2a" is necessary